internal/provider/ovirt/inventory.go

```diff
@@ -45,6 +45,18 @@ func (p *ovirtProvider) SetPlatformValuesInDBUpdates(
 	return nil
 }
 
+// GetActualSchedulableMasters returns the value of the SchedulableMasters cluster
+// property in case it was actively set through the API or 'false' otherwise.
+func (p *ovirtProvider) GetActualSchedulableMasters(cluster *common.Cluster) (bool, error) {
```

Comment:
why is it different from `func (p *baremetalProvider) GetActualSchedulableMasters(cluster *common.Cluster) (bool, error) {` ?

Reply:
In oVirt platform (and Vsphere most probably) we need to disable the scheduling of workloads on masters by default. 

This is because the oVirt provider always sets the compute replicas to 0 in the 'install-config.yaml' file to prevent the machine operator to provision additional compute machines at installation time (the compute nodes installed by the Assisted Installer will be `Nodes` only and not part of the `MachineSet`).

The side effect of doing this is that the generated manifests will enable the scheduling of workloads in the control plane (no compute nodes in the 'install-config.yaml'). Therefore, It does not matter how many nodes belong to the cluster, the scheduling of workloads on the control plane would be always enabled.